### ISSUE TITLE
Stuck at 0%

### ISSUE THREAD

**Issue by ShacharWeis** [openai/shap-e#64]
I cloned the repo and installed the packages needed. 
Running on a AMD Ryzen 9  laptop with an RTX3070 at 16gb RAM

I run a text-to-model sample and I see this:
 0%|          | 0/64 [00:00<?, ?it/s]

after 15 minutes it's still at 0%
Is it stuck or just really slow?

**Comment by Kabanosk** [comment]
I think that is because you run computation on CPU instead of GPU. Can you print the value of the`device` variable?

**Comment by ShacharWeis** [comment]
Yes, that was the problem. I installed torch with CUDA and now it works. 

Here is the fix:

pip installing torch
pip install torch torchvision torchaudio --index-url https://download.pytorch.org/whl/cu118

**Comment by BryanValc** [comment]
> Yes, that was the problem. I installed torch with CUDA and now it works.
> 
> Here is the fix:
> 
> pip installing torch pip install torch torchvision torchaudio --index-url https://download.pytorch.org/whl/cu118

For me it still doesn't work, I have an RTX 3060 12G and Ryzen 5 2600, isn't it because of the python version? I'm using Python 3.11.3, let me guys know what version are you using

**Comment by Kabanosk** [comment]
@BryanValc Can you please write what is value of `device` variable?
```python
device = 'cuda' if torch.cuda.is_available() else 'cpu'
print(device)
```

**Comment by BryanValc** [comment]
> @BryanValc Can you please write what is value of `device` variable?
> 
> ```python
> device = 'cuda' if torch.cuda.is_available() else 'cpu'
> print(device)
> ```

It keeps using cpu 
![image](https://github.com/openai/shap-e/assets/71149864/1d68ff16-6106-44f9-903d-b8c011cb9ccd)

**Comment by BryanValc** [comment]
@Kabanosk I did that, but it keeps using cpu 
![image](https://github.com/openai/shap-e/assets/71149864/3bb92fef-e485-4e8e-8819-9e24e7223dc0)

**Comment by ShacharWeis** [comment]
You have to uninstall torch and then install a CUDA version of torch. Then the variable value is "cude" 

Or, you can just not bother, because the outcome from this AI is complete garbage. Really. The examples they show here were carefully hand picked. It's a waste of time (in it's current state)

**Comment by BryanValc** [comment]
> You have to uninstall torch and then install a CUDA version of torch. Then the variable value is "cuda"
> 
> Or, you can just not bother, because the 3D models output from this AI are complete garbage. Really. The examples they show here were carefully hand picked. It's a waste of time (in it's current state). I played with it for an hour and gave up. Nothing it makes is even remotely unusable.

I tried uninstalling it, then running this command: pip install torch torchvision torchaudio --index-url https://download.pytorch.org/whl/cu118

But still nothing.. maybe you're right, it isn't much of a big deal as of now, it is like the beginning of stable diffusion

**Comment by Le0m1** [comment]
#123 i have a problem with GPU, but i can't fix this

**Comment by siddharth952** [comment]
`pip uninstall torch`
`pip install torch torchvision torchaudio --index-url https://download.pytorch.org/whl/cu118`
Then restart your kernel
